ui/app/src/utils/state.ts

```diff
@@ -29,11 +29,11 @@ export function getStoredGlobalMenuSiteViewPreference(user: string): 'grid' | 'l
 
 export function getStateMapFromLegacyItem(item: LegacyItem): ItemStateMap {
   return {
-    live: false,
+    live: item.endpoint === 'live',
     locked: false,
     modified: false,
```

Comment:
There must be a way of knowing these as the old ui shows the lock and pencil icons